configs/neox_arguments.md

```diff
@@ -261,6 +261,14 @@ Model Arguments
 
 
 
+- **layernorm_fusion**: bool
```

Comment:
why is this here?

Reply:
outstanding question. Will remove.

Reply:
Actually this belongs here. Just not in this commit. This is autogenerated by https://github.com/EleutherAI/gpt-neox/pull/1138/commits/e840a061611507f972f0d381eeaac48cc336b338. I think this is a results of our docs updater not working before